src/main/java/net/pms/store/item/RealFile.java

```diff
@@ -82,6 +84,25 @@ public RealFile(Renderer renderer, File file, String name) {
 		setLastModified(file.lastModified());
 	}
 
+	/**
+	 * Check if this this a new resource.
+	 *
+	 * @return true : File is an empty container. Upload by AV client is still missing. No need to check any formats yet.
+	 */
+	private boolean isUploadResource(File file, int type) {
+		if (type == Format.AUDIO || type == Format.VIDEO) {
+			try {
+				if (Files.size(file.toPath()) == UmsContentDirectoryService.EMPTY_FILE_CONTENT.length()) {
+					LOGGER.trace("isUploadResource true");
```

Comment:
I'm not sure if this logging is useful with no context, maybe it should mention the file it was looking for